templates/demoDynamoDB.yml

```
@@ -0,0 +1,59 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  demoDynamoDBTable: 
+    Type: 'AWS::DynamoDB::Table'
+    Properties: 
+      AttributeDefinitions: 
+        - 
+          AttributeName: "ISBN"
+          AttributeType: "S"
+        - 
+          AttributeName: "Title"
+          AttributeType: "S"
+        - 
+          AttributeName: "Description"
+          AttributeType: "N"
+      KeySchema: 
+        - 
+          AttributeName: "ISBN"
+          KeyType: "HASH"
+        - 
+          AttributeName: "Title"
+          KeyType: "RANGE"
+      ProvisionedThroughput: 
+        ReadCapacityUnits: "5"
+        WriteCapacityUnits: "5"
+      TableName: "demoTableName"
+      GlobalSecondaryIndexes: 
+        - 
+          IndexName: "demoGSI"
+          KeySchema: 
+            - 
+              AttributeName: "Title"
+              KeyType: "HASH"
+          Projection: 
+            NonKeyAttributes: 
+              - "Description"
+            ProjectionType: "INCLUDE"
+          ProvisionedThroughput: 
+            ReadCapacityUnits: "5"
+            WriteCapacityUnits: "5"
+      LocalSecondaryIndexes: 
+        - 
+          IndexName: "demoLSI"
+          KeySchema: 
+            - 
+              AttributeName: "ISBN"
+              KeyType: "HASH"
+            - 
+              AttributeName: "Description"
+              KeyType: "RANGE"
+          Projection: 
+            NonKeyAttributes: 
+              - "Title"
+            ProjectionType: "INCLUDE"
+Outputs:
+  TableName:
+    Value: 
+      Ref: "demoDynamoDBTable"
```

You can use short form syntax here:

```suggestion
      !Ref demoDynamoDBTable
```

---

Done